app/models/sequence.rb

```diff
@@ -99,7 +99,8 @@ def export
                                         :logo,
                                         :display_title,
                                         :thumbnail_url,
-                                        :runtime
+                                        :runtime,
+                                        :background_image
```

Comment:
I think this has the same issue of not being imported correctly.

Reply:
@scytacki Thanks for catching that. I've fixed it.